website/docs/r/wiki.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_wiki"
+description: |-
+  Manages Wikis within Azure DevOps project.
+---
+
+# azuredevops_wiki
+
+Manages Wikis within Azure DevOps project.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  name = "My Awesome Project"
+  description  = "All of my awesomee things"
+}
+
+resource "azuredevops_git_repository" "repository" {
+  project_id = azuredevops_project.project.id
+  name       = "My Awesome Repo"
+  initialization {
+    init_type = "Clean"
+  }
+}
+
+resource "azuredevops_wiki" "test" {
+  name = "project wiki "
+  project_id = azuredevops_project.project.id
+  type = "projectWiki"
+}
+
+resource "azuredevops_wiki" "test2" {
+  name = "additional  wiki in repo"
+  project_id = azuredevops_project.project.id
+  repository_id = azuredevops_git_repository.repository.id
+  version = "main"
+  type = "codeWiki"
+  mappedpath = "/"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project_id` - (Required) The ID of the Project.
+- `name` - (Required) The name of the Wiki.
+- `type` -  (Required) The type of the wiki. Possible values are `codeWiki`, `projectWiki`.
```

Comment:
Since the project type wiki cannot be delete when the project not deleted, we should add a notice here:
```suggestion
- `type` -  (Required) The type of the wiki. Possible values are `codeWiki`, `projectWiki`.

~> **NOTE:** Project type wiki can only be deleted together with the project.

```